database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -700,6 +700,41 @@ CREATE TRIGGER review_assignment_validate_section_trigger
     WHEN (NEW.trigger IS NULL AND OLD.trigger IS NULL)
     EXECUTE FUNCTION public.enforce_asssigned_section_validity ();
 
+-- FUNCTION to set `available_sections` on given review_assignments based on other assignments
```

Comment:
```suggestion
-- FUNCTION to return `available_sections` on given review_assignments based on other assignments
```
We never see this as a column in postgres, on in GraphQL, so we should make that clear by emphasising its a function not a column

Reply:
Done